src/components/ExportDataModal.js

```diff
@@ -17,9 +17,9 @@ import * as Utils from 'src/libs/utils'
 import validate from 'validate.js'
 
 
-const InfoTile = ({ infoStyle, content, iconName }) => {
+const InfoTile = ({ infoStyle, content, iconName, iconColor }) => {
```

Comment:
Since `infoStyle` and `iconColor` are connected, can we wrap them together into a single object that's just passed in as a prop once?